contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -640,7 +566,6 @@ contract StakeManager is IStakeManager {
                     valPow = ValidatorShare(validator.contractAddress)
                         .udpateRewards(_reward, stakePower);
                 } else {
-                    //TODO: check for div leaks in rewards amount
```

Comment:
So are the leaks being accounted for?

Reply:
These are minor leaks for smaller decimal points. Adding it into linear issue, as of now it doesn't seem to be problematic.